package.json

```
@@ -189,6 +189,7 @@
 		"lint-css": "wp-scripts lint-style '**/*.scss'",
 		"lint-css:fix": "npm run lint-css -- --fix",
 		"lint-types": "tsc",
+		"lint-md": "wp-scripts lint-md '**/*.md'",
```

I think 
```suggestion
		"lint-md": "wp-scripts lint-md",
```
should be enough since the code provides defaults.